internal/server/users.go

```diff
@@ -48,7 +48,7 @@ func (a *API) CreateUser(c *gin.Context, r *api.CreateUserRequest) (*api.CreateU
 	infraProvider := access.InfraProvider(c)
 
 	// infra identity creation should be attempted even if an identity is already known
-	identities, err := access.ListIdentities(c, user.Name, 0, nil, &models.Pagination{Limit: 2})
+	identities, err := access.ListIdentities(c, user.Name, 0, nil, false, &models.Pagination{Limit: 2})
```

Comment:
This will error when a user is created with the name "connector". 

That is actually a good thing though, because the connector shouldn't get assigned a password. Maybe we should add a check that the user name is not "connector" for a better error message though.